testing/src/main/java/io/airlift/testing/Closeables.java

```diff
@@ -11,36 +8,36 @@ private Closeables()
     {
     }
 
-    public static void closeQuietly(Closeable... closeables)
+    public static void closeQuietly(AutoCloseable... closeables)
     {
         if (closeables == null) {
             return;
         }
-        for (Closeable closeable : closeables) {
+        for (AutoCloseable closeable : closeables) {
             try {
                 if (closeable != null) {
                     closeable.close();
                 }
             }
-            catch (IOException | RuntimeException ignored) {
+            catch (Exception ignored) {
```

Comment:
you shouldn't ignore InterruptedException

Reply:
Thanks. Updated.